manifests/plugin/discovery.pp

```diff
@@ -4,5 +4,6 @@
 #
 class foreman::plugin::discovery {
   foreman::plugin {'discovery':
+    has_api => true,
```

Comment:
We would need to go over all plugins and determine this. This is just an example. Packaging should tell us easily what the value is.